### ISSUE TITLE
Suggested items to add to member profile (age/birthyear, facebook, twitter, etc.)

### ISSUE THREAD

**Issue by rfay** [warmshowers/Warmshowers.org#206]
There are a few things that should probably be added to the user profile, form, csv, and views integration of users.
- Internet/WIFI available
- Age/birthdate
- Current traveler location lat/lon
- Sex (hard to put down with couples accounts)
- Facebook, twitter, skype
- House rules (per #185)
- Nearest grocery (and perhaps closing time?)
- Nearest restaurant
- Profession
- Current trip information (style, length, mode)
- Recent home location/job/etc
- Currently traveling?
- #406 is for responsiveness

Note that when these are added as fields on user or in the form additional support must be provided:
- Views integration
- Services integration
- Presentation on the profile page
- Presentation on search results page

It's worth adding some of these things to an "extra" fieldset in the form (extra fieldset for host, extra fieldset for traveler). I think we can show them successfully on the profile - the problem is making the form overwhelming. 

Possibilities:
- Don't expose all of these in the initial signup. Instead add them to the edit form. 
- Send an email the day after or something asking to fill in more. 
- Don't allow to set as "available" if they haven't filled in hosting info
- Don't allow to set as "traveler" if they haven't filled in traveler info
- Don't allow to request hospitality if profile is inadequate

**Comment by deKolonel** [comment]
Right, please let's make sure not to be as many other sites with an awful lot of uninteresting information that you neither want to enter nor want to read... What we have is pretty much what you need to know as a guest and the profile text should tell the story. Remember we are not hotels; we don't need to sell ourselves... Looking at the list of suggestions nothing is mandatory... Another thing to keep in mind is that the people who take/took the time to set up a decent profile in general have all relevant information and in general also keep it up-to-date and for those who have a poor or not up-to-date profile, that definitely won't change with additional fields. The only advantage I see for having some additional fields is being able to filter on those fields when searching for hosting or company...

**Comment by kalpaitch** [comment]
The above is true, I have met people who have done some very good things with their profiles as is. Just adding detailed descriptions.  Too much information and it starts to become like a hotel service. 

I think services like wifi/Internet shouldn't be included.

I think house rules is a good idea but can be put in the description. 

Current trip information is a good idea. But I believe most of the other fields would be more than is necessary, would make it seem too formal and bloated, one of the reasons I dislike using CS.

**Comment by jackturner7678** [comment]
I would disagree with wifi/internet.  Without it warmshowers doesn't exist.

**Comment by kalpaitch** [comment]
I'm not saying I wish there was no wifi :) just that it shouldn't be searchable criteria, it's an extra service that shouldn't disadvantage a host for not having it.

**Comment by kalpaitch** [comment]
It's not that difficult too find Internet cafes, data sims or other wifi hot spots if you need it.

**Comment by deKolonel** [comment]
Correct

Sent from iPhone

**Comment by jackturner7678** [comment]
It's always important to me and it is always asked for when we host.  It's nice to ketch up emails in the evening and make arrangement for the next few days without having to drag out everything at a coffee shop.

**Comment by klzig** [comment]
A feature I've planned from the beginning for the WP app is the ability to filter searches based on hosting services.  If we agree that is a good thing, I don't know why we'd exclude wifi availability from the filter any more than we would kitchen access or language spoken.  For a cyclist using a phone app (or worse, a laptop/tablet with no cellular), it might matter a lot whether a host offers wifi.  And for others not using AT&T for cell coverage it might not matter at all :)  I'd personally rank wifi above most of the things we currently advertise for hosts.  Because I'm on AT&T.

Volunteer hosts are awesome and Warm Showers wouldn't exist without them, but it seems to me the reason the service exists is cyclists who are currently on tour.  I think we should prioritize features based on their needs first.  There are many things a host can do if they want to lure in more cyclists.  Like move to a town on a major bike route :). Take a look at my dismal hosting percentages...

**Comment by martinmemartin** [comment]
For what it matters, having hosted nearly 150 bike tourists, I have never
had a guest NOT ask if I have wifi. In fact, it was because of this
constant request that I went out and GOT wifi. ;-)

Mark

**Comment by deKolonel** [comment]
Nobody ever said Wifi is not handy or even not prefered... :-) Of course Wifi is handy! The discussion is/was whether adding it as a search criteria is good or not. Some say it is, some say it is not...

**Comment by martinmemartin** [comment]
I suspect that the good / not good discussion will continue. I would say,
yes, add it, if for no other reason than I wouldn't be asked as much. ;-)

**Comment by rfay** [comment]
This doesn't need any more debate here. Everybody interested in feature priorities should spend a half hour looking at https://www.warmshowers.org/feature_poll and seeing the massive community input there. The most interesting thing was that age/birthyear, which has been regularly requested, was soundly rejected by everybody who cared.

**Comment by deKolonel** [comment]
I agree on both. Majority counts here (or if not sensitivity (around age for instance)).

Focus should now be on how to deliver all this in the profile. I think you added some good suggestions; only a few fields mandatory and we can request adding more if not initially added.

I think it would also be reasonable to have search functionality as such that it will first give the results for exact match and then other results based on nearby even there is no exact match.

**Comment by kalpaitch** [comment]
Referencing https://github.com/warmshowers/Warmshowers.org/issues/504 as this ticket would be an appropriate time (from a technical standpoint) to consider other profile updates.
